griptape/drivers/prompt_model/bedrock_titan_prompt_model_driver.py

```diff
@@ -67,14 +67,17 @@ def prompt_stack_to_model_params(self, prompt_stack: PromptStack) -> dict:
             }
         }
 
-    def process_output(self, response_body: str | bytes) -> TextArtifact:
+    def process_output(self, output: list[dict] | str | bytes) -> TextArtifact:
         # When streaming, the response body comes back as bytes.
-        if isinstance(response_body, bytes):
-            response_body = response_body.decode()
+        if isinstance(output, str or bytes):
```

Comment:
I'm not sure if this syntax does what we think it does. Let's rewrite to `isinstance(output, str) or isinstance(output, bytes)`